docs/quickstart/quickstart_chains/ethereum-opensea.md

```diff
@@ -0,0 +1,533 @@
+# Ethereum Quick Start - Opensea (Medium)
+
+Welcome to our comprehensive step-by-step guide dedicated to constructing a Subquery indexer tailored for the OpenSea marketplace. OpenSea has emerged as a global epicenter for NFTs, establishing itself as a vibrant ecosystem for creators, collectors, and traders alike. Whether you're a developer seeking to construct applications that harness the power of NFT data or an enthusiast eager to immerse yourself in the realm of digital collectibles, this guide is designed to seamlessly lead you through the steps of configuring your personal OpenSea Subquery indexer.
```

Comment:
It feels a bit too much: "Whether you're a developer seeking to construct applications that harness the power of NFT data or an enthusiast eager to immerse yourself in the realm of digital collectibles...". In my opinion this fits better to the marketing piece, the article

Reply:
Yes, this is a developer guide. Keep it short and simple, save the above for marketing articles